app/models/patient.rb

```diff
@@ -55,12 +66,16 @@ class Patient
   field :household_size_adults, type: Integer
   field :insurance, type: String
   field :income, type: String
+  field :special_circumstances, type: Array, default: []
```

Comment:
I think in practice it winds up being either. Some of 'em are special circumstances for the pregnancy (a health diagnosis of some kind) and some of 'em are a patient's circumstances that give us more context into additional ways we can help them (per my understanding, stuff on the order of 'by the way, here's our person at a shelter', etc). But since a goal of DCAF is to center patients in abortion care I think this a step toward reflecting that in our codebase.